WeatherApp/WeatherApp/App/DI/Dependencies.swift

```diff
@@ -39,14 +41,30 @@ class Dependencies: DependenciesProtocol {
         WeatherService(client: weatherClient)
     }()
 
+    private lazy var dataService: DataServiceProtocol = {
+        DataService()
+    }()
+
     private lazy var weatherRepository: WeatherRepositoryProtocol = {
         WeatherRepository(weatherService: weatherService)
     }()
 
+    private lazy var dataRepository: DataRepositoryProtocol = {
+        DataRepository(dataService: dataService)
+    }()
+
     lazy var getWeatherUseCase: GetWeatherUseCaseProtocol = {
         GetWeatherUseCase(weatherRepo: weatherRepository)
     }()
 
+    lazy var getCitiesUseCase: GetCitiesUseCaseProtocol = {
+        GetCitiesUseCase(dataRepo: dataRepository)
+    }()
+
+    lazy var storeCitiesUseCase: StoreCitiesUseCaseProtocol = {
+        StoreCitiesUseCase(dataRepo: dataRepository)
+    }()
+
```

Comment:
can this not be public? If you already fixed this, ignore it

Reply:
They can't be private.

Property 'getCitiesUseCase' must be declared internal because it matches a requirement in internal protocol 'UseCaseDependenciesProtocol'